[bitnami/cosign] login fails due to permission denied

### Name and Version

bitnami/cosign:2.0.0

### What architecture are you using?

amd64

### What steps will reproduce the bug?

I am trying to use bitnami/cosign image to sign container images stored in quay.io based registry. Performing prerequisite login command failed due to permission issue:

```
[mahnat@DESKTOP ~]# docker run -it bitnami/cosign:2.0.0 login $REGISTRY -u $USER -p $PASSWORD
Error: mkdir /.docker: permission denied
main.go:74: error during command execution: mkdir /.docker: permission denied
[mahnat@DESKTOP ~]#
```

Login command attempts to create `~/.docker/config.json` file but user 1001 is homeless so subsequently the creation of `/.docker` directory fails for obvious reasons. Dockerfile needs additional stuff to be added as user 1001 needs a writable home directory. WORKDIR and USER declarations are not sufficient for `cosign login`.

### What is the expected behavior?

Login successfully creates ~/.docker/config.json file required for other cosign operations (sign, verify, etc.) and login finish with exit code 0.

Output for cosign login should be similar to this:
```
[mahnat@DESKTOP ~]# cosign login $REGISTRY -u $USER -p $PASSWORD
auth.go:203: logged in via /home/mahnat/.docker/config.json
[mahnat@DESKTOP ~]#
```

### What do you see instead?

```
[mahnat@DESKTOP ~]# docker run -it bitnami/cosign:2.0.0 login $REGISTRY -u $USER -p $PASSWORD
Error: mkdir /.docker: permission denied
main.go:74: error during command execution: mkdir /.docker: permission denied
[mahnat@DESKTOP ~]#
```

### Additional information

_No response_

--- a/bitnami/cosign/2/debian-11/Dockerfile
+++ b/bitnami/cosign/2/debian-11/Dockerfile
@@ -38,6 +38,7 @@ RUN apt-get autoremove --purge -y curl && \
     apt-get clean && rm -rf /var/lib/apt/lists /var/cache/apt/archives
 RUN chmod g+rwX /opt/bitnami
 RUN mkdir -p "/cosign-keys" && chmod -R 777 "/cosign-keys"
+RUN mkdir -p "/.docker" && chmod g+rwX "/.docker"
 RUN mkdir /.sigstore && chmod g+rwX /.sigstore
 
 ENV APP_VERSION="2.0.2" \

